src/database/seeders/20210726151520-mock-data-users.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
Убрать неиспользуемый параметр
```suggestion
  up: async (queryInterface) => {
```

Reply:
исправлен